tests/test_elastic.py

```diff
@@ -763,3 +770,80 @@ def test_multiple_migrations_with_delete(self):
             var = self.es.indices.get_aliases()
             assert_equal(var[settings.ELASTIC_INDEX + '_v{}'.format(n + 1)]['aliases'].keys()[0], settings.ELASTIC_INDEX)
             assert not var.get(settings.ELASTIC_INDEX + '_v{}'.format(n))
+
+class TestSearchFiles(SearchTestCase):
+
+    def setUp(self):
+        super(TestSearchFiles, self).setUp()
+        self.node = ProjectFactory(is_public=True, title='Otis')
+        self.osf_storage = self.node.get_addon('osfstorage')
+        self.root = self.osf_storage.get_root()
+
+    def test_search_file(self):
+        self.root.append_file('Shake.wav')
+        find = query_file('Shake.wav')['results']
+        assert_equal(len(find), 1)
+
+    def test_delete_file(self):
+        file_ = self.root.append_file('I\'ve Got Dreams To Remember.wav')
+        find = query_file('I\'ve Got Dreams To Remember.wav')['results']
+        assert_equal(len(find), 1)
+        file_.delete()
+        find = query_file('I\'ve Got Dreams To Remember.wav')['results']
+        assert_equal(len(find), 0)
+
+    def test_add_tag(self):
+        file_ = self.root.append_file('That\'s How Strong My Love Is.mp3')
+        from website.models import Tag
```

Comment:
Does this import need to be inside of the test? Since it's being used in at least two tests it would be nicer to just import it once.